### PR TITLE
bump gradle version to 6.2.2

### PR DESCRIPTION
**Pull request by martinbonnin**

### CODE REVIEW

**Comment by tasomaniac** [comment]
In this PR, shall we try removing artifact renaming for KMP and use official KMP artifact naming conventions. Just to see if tests work.

**Comment by tasomaniac** [comment]
> Execution failed for task ':apollo-android-support:checkstyle'.
> Unable to create Root Module: config {/home/travis/build/apollographql/apollo-android/checkstyle.xml}, classpath {null}.

**Comment by martinbonnin** [comment]
> In this PR, shall we try removing artifact renaming for KMP and use official KMP artifact naming conventions. Just to see if tests work.

I've done this and bumped the minimum Gradle version to 6.0

File: gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-5.6.4-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.2.2-all.zip
```
Comment (tasomaniac) [inline gradle/wrapper/gradle-wrapper.properties]: If you run the `gradle wrapper` task, there might be some updates to jar and executable files.

File: checkstyle.xml [hunk 2]
```diff
@@ -47,12 +51,6 @@
       <property name="processJavadoc" value="true"/>
     </module>
 
-
-    <!-- Checks for Size Violations.                    -->
-    <!-- See http://checkstyle.sf.net/config_sizes.html -->
-    <module name="LineLength">
-      <property name="max" value="140"/>
-    </module>
```
Comment (martinbonnin) [inline checkstyle.xml]: checkstyle is bundled with Gradle. In newer versions, `cacheFile` and `LineLength` have been moved one level above.

**Pull request merged**